src/main/java/com/searchservice/app/domain/dto/table/SchemaDto.java

```diff
@@ -0,0 +1,23 @@
+package com.searchservice.app.domain.dto.table;
+
+public enum SchemaDto {
```

Comment:
Pls change this enum class name to **_SchemaLabel_**

Reply:
ok